### PR TITLE
Add EIP: Math checking in EVM

### PR DESCRIPTION
**Pull request by KSST88**
#When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by KSST88** [comment]
#6958

**Comment by KSST88** [comment]
#6958

**Review by KSST88 — COMMENTED** [review]
#6958

**Comment by KSST88** [comment]
#6958 #6958

File: EIPS/eip-NaN.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+eip: .nan
```
Comment (abcoathup) [inline EIPS/eip-NaN.md:2]:
```suggestion
eip: 6958
```
Issuing EIP number 6958 as this is the PR number.  Please also update the filename

**Pull request closed without merging**